reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,322 @@
+[PowerShell Support Lifecycle](/powershell/scripting/powershell-support-lifecycle).
+
+## Installing PowerShell 7
+
+For flexibility and to support the needs of IT, DevOps engineers, and developers, there are several
+options available to install PowerShell 7. In most cases, the installation options can be reduced to
+the following methods:
+
+- Deploy PowerShell using the [MSI package](/powershell/scripting/install/installing-powershell-core-on-windows#installing-the-msi-package)
+- Deploy PowerShell using the [ZIP package](/powershell/scripting/install/installing-powershell-core-on-windows#installing-the-zip-package)
+
+> [!NOTE]
+> The MSI package can be deployed and updated with management products such as Microsoft System Center
+> Configuration Manager (SCCM). Download the packages from [GitHub Release page](https://github.com/PowerShell/PowerShell/releases).
+
+## Using PowerShell 7 side-by-side with Windows PowerShell 5.1
+
+PowerShell 7 is designed to coexist with Windows PowerShell 5.1. The following features ensure that
+your investment in PowerShell is protected and your migration to PowerShell 7 is simple.
+
+- Separate installation path and executable name
+- Separate PSModulePath
+- Separate profiles for each version
+- Improved module compatibility
+- New remoting endpoints
+- Group policy support
+- Separate Event logs
+
+### Separate installation path and executable name
+
+PowerShell 7 installs to a new directory, enabling side-by-side execution with Windows PowerShell
+5.1.
+
+Install locations by version:
+
+- Windows PowerShell 5.1: `C:\Windows\System32\WindowsPowerShell\v1.0`
+- PowerShell Core 6.x: `%programfiles%\PowerShell\6`
+- PowerShell 7: `%programfiles%\PowerShell\7`
+
+The new location is added to your PATH allowing you to run both Windows PowerShell 5.1 and
+PowerShell 7. If you're migrating from PowerShell Core 6.x to PowerShell 7, PowerShell 6 is removed
+and the PATH replaced.
+
+In Windows PowerShell, the PowerShell executable is named `powershell.exe`. In version 6 and above,
+the executable is named `pwsh.exe`. The new name makes it easy to support side-by-side execution of
+both versions.
+
+### Separate PSModulePath
+
+By default, Windows PowerShell and PowerShell 7 store modules in different locations. PowerShell 7,
+combines those locations in the `$Env:PSModulePath` environment variable. When importing a module by
+name, PowerShell checks the location specified by `$Env:PSModulePath`. This allows PowerShell 7 to
+load both Core and Desktop modules.
+
+| Version |                System-wide location                |           User-specific location            |
+| ------- | -------------------------------------------------- | ------------------------------------------- |
+| PS 5.1  | `%WINDIR%\system32\WindowsPowerShell\v1.0\Modules` | `$HOME\Documents\WindowsPowerShell\Modules` |
+| PS 7.0  | `$PSHOME\Modules`                                  | `$HOME\Documents\PowerShell\Modules`        |
```

Comment:
There's also the shared location in ProgramFiles which you already have below, should probably be in this table.

Reply:
Fixed